src/serializers.jsx

```diff
@@ -0,0 +1,29 @@
+const serializeWordDefinition = (response) => {
+  let simplifiedResponse = {};
+  simplifiedResponse.word = response[0].word;
+  simplifiedResponse.sections = [];
+  response?.forEach((each) => {
```

Comment:
Rename `each`. The name should give an idea of what data the variable holds. Sure I've said this before. Please be more watchful while naming variables.

Rename this everywhere.